src/Symfony/Bundle/FrameworkBundle/Command/WorkerListCommand.php

```diff
@@ -0,0 +1,26 @@
+<?php
+
+namespace Symfony\Bundle\FrameworkBundle\Command;
+
+use Symfony\Component\Console\Input\InputInterface;
+use Symfony\Component\Console\Output\OutputInterface;
+
+class WorkerListCommand extends ContainerAwareCommand
+{
+    protected function configure()
+    {
+        $this
+            ->setName('worker:list')
+            ->setDescription('List available workers.')
+        ;
+    }
+
+    protected function execute(InputInterface $input, OutputInterface $output)
+    {
+        $workers = $this->getContainer()->getParameter('worker.workers');
```

Comment:
Injecting an array isn't reliable too, because array can be of any structure. Still, I also like to have commands as a service. Do we need to check somewhere if `worker.workers` has the correct structure?